src/case-swapping.js

```diff
@@ -15,7 +15,17 @@
  * @return {string}
  */
 function swapCase(phrase) {
-  // write code here
+  let caseSwappingPhrase = '';
+
+  for (let i = 0; i < phrase.length; i++) {
+    if (phrase[i] === phrase[i].toLowerCase()) {
+      caseSwappingPhrase = caseSwappingPhrase + phrase[i].toUpperCase();
```

Comment:
There is a short entry
```suggestion
      caseSwappingPhrase += phrase[i].toUpperCase();
```